src/helpers/misc/Animal.re

```diff
@@ -89,8 +89,8 @@ let getTokenId: t => TokenId.t =
 let getNameFromId: string => string =
   animalId =>
     switch (animalId) {
-    | "0" => "Andy"
-    | "1" => "Simon"
+    | "0" => "Simon"
+    | "1" => "Andy"
```

Comment:
This is here by accident, it is from another PR: https://github.com/wild-cards/ui/pull/80/files